sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -2348,7 +2348,7 @@ class CollectionExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper
 
         // SQL array indices start at 1 exception throws for both mode.
         expr = ElementAt(array, Literal(0))
-        val errMsg = "SQL array indices start at 1"
+        val errMsg = "The index 0 is out of bounds. The array has 3 elements."
```

Comment:
The test `SPARK-33386: element_at ArrayIndexOutOfBoundsException` checks errors only. Should we move it to `QueryExecutionAnsiErrorsSuite` or/and `QueryExecutionErrorsSuite`?